### PR TITLE
ci: don't set DENO_CANARY when releasing, attempt 2

### PR DESCRIPTION
**Pull request by piscisaureus**

### CODE REVIEW

**Review by bartlomieju — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -194,6 +194,12 @@ jobs:
           cat >> ~/.bash_profile << ___
             cd "$(pwd)"
 
+            # Configure canary build.
+            if [ "$DENO_CANARY" == "true" ]
```
Comment (ry) [inline .github/workflows/ci.yml:198]: @piscisaureus Shouldn't this have parens?

Comment (piscisaureus) [reply .github/workflows/ci.yml:198]: With parens, that would (try to) run the command DENO_CANARY right?

Comment (ry) [reply .github/workflows/ci.yml:198]: Ah right. Nevermind.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -194,6 +194,12 @@ jobs:
...
+            then
+              export DENO_CANARY=true
```
Comment (maximousblk) [inline .github/workflows/ci.yml:200]: environment variables in github actions are not persisted across steps by default. you'll have to use a special environment variable `GITHUB_ENV`

```sh
echo "DENO_CANARY=true" >> $GITHUB_ENV
```

more info: https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable

**Pull request merged**